packages/api/src/controllers/access-control.ts

```diff
@@ -225,17 +225,20 @@ app.post(
 
     let config: Partial<GateConfig> = {};
 
-    if (user.viewerLimit) {
-      config.user_viewer_limit = user.viewerLimit;
-      config.user_id = user.id;
+    let concurrentViewers = DEFAULT_MAX_CONCURRENT_VIEWERS;
+    let userProduct = products[user.stripeProductId];
+
+    if (userProduct.concurrentViewers) {
+      concurrentViewers = userProduct.concurrentViewers;
     }
 
-    if (user.createdAt > HACKER_DISABLE_CUTOFF_DATE) {
-      if (isFreeTierUser(user)) {
-        config.rate_limit = MAX_ALLOWED_VIEWERS_FOR_HACKER_TIER_PER_NODE;
-      }
+    if (user.viewerLimit) {
+      concurrentViewers = user.viewerLimit;
```

Comment:
Now that I see this, since we already have the `viewerLimit` field on users WDYT of using the same name on the plans? Feels unnecessary to use a different name, even if the existing one is not ideal, and make it look like they mean different things (especially since we don't usually have a lot of documentation for internal stuff)

Reply:
Make sense! Changing everything to `viewerLimit`